snapcraft/file_utils.py

```diff
@@ -334,8 +334,9 @@ def calculate_hash(path: str, *, algorithm: str) -> str:
 def get_tool_path(command_name: str) -> str:
     """Return the path to the given command
 
-    Return a path to command_name, if Snapcraft is running out of the snap or out
-    of Docker, it ensures it is using the one in the snap, not the host.
+    Return a path to command_name, if Snapcraft is running out of the snap
+    or in legacy mode (snap or sources), it ensures it is using the one in
```

Comment:
Hmm, not sure I'd call it legacy mode, but regardless, I assume you mean to say "(deb or sources)" here?

Reply:
considering the snap will be the standard way we want to promote wherever your are, yes, all others are indeed legacy from that point of view

Reply:
fwiw, I am not changing the text here to wait another hour or so for tests to run, that might make us miss a release window for this week.